Interstation-Two-WW2/code/modules/WW2/train/lever.dm

```diff
@@ -77,13 +77,20 @@
 		if (findtext(what, "Start Train"))
 			what = replacetext(what, " (Start Train)", "")
 			switch (what)
-				if ("Push Forward")
-					train_controller.start_moving("FORWARDS")
-					visible_message("<span class = 'danger'>[user] pushes the train lever forward, starting the train!</span>")
-				if ("Pull Back")
-					train_controller.start_moving("BACKWARDS")
-					visible_message("<span class = 'danger'>[user] pulls back the train lever, starting the train!</span>")
-
+				var/list/hear = get_mobs_or_objects_in_view(world.view,src)
```

Comment:
You guys could add a Choo Choo sound or something like the train tracks starting up.